### DISCUSSION TITLE
How to use SignalR with Shiny.

### DISCUSSION THREAD

**Discussion by Vuyiswamb** [shinyorg/shiny#821] [Q&A]
Good Day All 
 
I am trying to use Shiny for Backgrounding . I have SignalR code that runs from the App.cs file , This obviously works when the app is opened. So now i want to move the code to the background Service.  So basically in my SignalR Code i show Local Notification. i have attached a simple example . i want my SignalR code to be fired and give me a notification on the foreground when the received event received something 

Example Link http://www.yowfun.com/eg/ShinyExample.zip

Thanks

**Comment by aritchie** [comment]
I think you have the wrong idea of how Shiny works.  You only get a small amount of time to execute periodic or triggered tasks in the background.  SignalR is not going to be running indefinitely.  If you need to use real time notifications in the background, you need to use Shiny Push

**Comment by Vuyiswamb** [comment]
Thank you very much , let me readup on Shiny push. am i right to say Shiny.Push is still your creation with the link below ? 

https://www.nuget.org/packages/Shiny.Push/ 

Thanks
